tools/please_go/please_go.go

```diff
@@ -179,14 +180,14 @@ var subCommands = map[string]func() int{
 	},
 	"package_info": func() int {
 		pi := opts.PackageInfo
-		if err := packageinfo.WritePackageInfo(pi.ImportPath, "", pi.Pkg, "", pi.ImportMap, pi.Complete, os.Stdout); err != nil {
```

Comment:
is the idea here that we never generate a package in the end that is incomplete? i.e. we sometimes call `go tool compile` without `-complete` but we end up completing it later in whatever this refers to (because we built the assembly in or whatever)

Reply:
Ah no actually I just noticed that `complete` wasn't used in `WritePackageInfo`! I actually forgot about that bit and didn't put a great deal of thought into it. I'm not too sure how it works to be honest, is it to do with getting non-go files into the pkg info? I'll leave it up to your judgement! If we delete it I'll remove the option from the struct too.

Reply:
Hmmmm I'm not sure it's even particularly useful - I thought it was on `Package` but can't see it. Happy to rip it out.